docs/pages/guides/style.md

```diff
@@ -209,7 +211,6 @@ inspect and undo changes in git.
 ```toml
 [tool.ruff]
 src = ["src"]
-exclude = []
```

Comment:
Only required if you have a build folder (pypa/build, scikit-build-core), but not generally required, so dropping it here.